reana_client/cli/utils.py

```diff
@@ -315,10 +294,10 @@ def output_user_friendly_logs(workflow_logs, steps):
     :param steps: List of steps to show logs for.
     """
     # Colors matching the status colors of REANA-UI
-    status_to_color_mapping = {
-        "running": "bright_blue",
-        "finished": "green",
-        "failed": "red",
+    status_to_msg_type = {
+        "running": "info",
+        "finished": "success",
+        "failed": "error",
```

Comment:
*suggestion*: move it to constant dict to `config.py` as `JOB_STATUS_TO_MSG_TYPE`

Reply:
Done in https://github.com/reanahub/reana-client/pull/553/commits/5958a7db472f5a71d7ac48e39ea65d8c90af1873